homeassistant/components/deconz/light.py

```diff
@@ -113,11 +125,11 @@ class DeconzBaseLight(DeconzDevice, LightEntity):
 
     TYPE = DOMAIN
 
-    def __init__(self, device, gateway):
+    def __init__(self, device: Group | Light, gateway: DeconzGateway) -> None:
         """Set up light."""
         super().__init__(device, gateway)
 
-        self._attr_supported_color_modes = set()
+        self._attr_supported_color_modes: set = set()
```

Comment:
I suggest doing this retype in the class attribute, as it is more clear it is being re-typed at that point.

Reply:
I couldn't move the whole statement do you mean I should just move the retype and not the assignment?

Reply:
Nah this is fine, I needed to have a better look myself before I placed this comment 😵‍💫